src/components/AccountDrawer/MiniPortfolio/Activity/parseRemote.test.tsx

```diff
@@ -0,0 +1,392 @@
+import { act, renderHook } from '@testing-library/react'
+import { WETH9 } from '@uniswap/sdk-core'
+import { DAI } from 'constants/tokens'
+import ms from 'ms'
+
+import {
+  MockClosedUniswapXOrder,
+  MockMoonpayPurchase,
+  MockNFTApproval,
+  MockNFTApprovalForAll,
+  MockNFTPurchase,
+  MockNFTReceive,
+  MockNFTTransfer,
+  MockOpenUniswapXOrder,
+  MockOrderTimestamp,
+  MockRecipientAddress,
+  MockRemoveLiquidity,
+  MockSenderAddress,
+  MockSwapOrder,
+  MockTokenApproval,
+  MockTokenReceive,
+  MockTokenSend,
+  MockTokenTransfer,
+} from './fixtures/activity'
+import { parseRemoteActivities, useTimeSince } from './parseRemote'
+
+describe('parseRemote', () => {
+  beforeEach(() => {
+    jest.useFakeTimers()
+  })
+  describe('parseRemoteActivities', () => {
+    it('should not parse open UniswapX order', () => {
+      const result = parseRemoteActivities(jest.fn(), [MockOpenUniswapXOrder])
+      expect(result).toEqual({})
+    })
+    it('should parse closed UniswapX order', () => {
+      const result = parseRemoteActivities(jest.fn(), [MockClosedUniswapXOrder])
+      expect(result?.['someHash']).toEqual({
```

Comment:
These would be excellent candidates for snapshot testing, because they verify that the parsed object doesn't change. Instead of manually entering what it should equal, why not just `expect(result?.['someHash']).toMatchSnapshot()`?

I think this whole file could be simplified with snapshot testing, as you're testing that the parsing isn't changing over time - it's less relevant exactly how it is parsed.

Reply:
yeah i agree